### ISSUE TITLE
Reverse definitions of size_frac_low and size_frac_up

### ISSUE THREAD

**Issue by cpavloud** [emo-bon/observatory-profile#7]
@09012000-tosca
In the [logsheet_schema_extended.csv](https://github.com/emo-bon/observatory-profile/blob/main/logsheet_schema_extended.csv)
change the definitions of size_frac_low and size_frac_up. They should be reversed (as mentioned also in the ENA checklist)

**Comment by cpavloud** [comment]
Discussed also [here](https://github.com/emo-bon/observatory-esc68n-crate/issues/5)

**Comment by laurianvm** [comment]
Hi @marc-portier, @bulricht, 
Can you double check the data quality control flow for any consequences of this swap?

**Comment by kmexter** [comment]
does not affect the QC. @cpavloud do you want any other types of checks run on these values, e.g. that > or < is not being used - or even that we just correct that to NA?

**Comment by cpavloud** [comment]
In the size_frac_low and size_frac_up columns there should be no > and <. 
If there is for example **size_frac_low >20**, then this should be converted to **size_frac_low 20** and **size_frac_up NA**
But I think this should be done at the data provider side of things, no?

**Comment by kmexter** [comment]
yes sure, I was asking if you wanted us to add a check for that in the QC (currently there is just a check on whether the values in these columns are a float or not, and as the presence of "<" will of course make it not-a-float, there will be an error report anyway)

**Comment by cpavloud** [comment]
Yeah, we should probably add a check for this too.

**Comment by laurianvm** [comment]
addition of QC check being handled in https://github.com/emo-bon/data-quality-control-action/issues/8
